telemetry/definitions/commonDefinitions.json

```diff
@@ -119,14 +119,39 @@
             ]
         },
         {
-          "name": "regionId",
-          "type": "string",
-          "description": "The ID of the region that was selected"
+            "name": "regionId",
+            "type": "string",
+            "description": "The ID of the region that was selected"
+        },
+        {
+            "name": "partitionId",
```

Comment:
not used? also not needed

Reply:
partitionId was already present, but didn't have the indentation formatting

Reply:
Ah I see it's used in setPartition, that's new so didn't know it was there